.github/autobuild/linux_deb.sh

```diff
@@ -6,13 +6,22 @@ if [[ ! ${JAMULUS_BUILD_VERSION:-} =~ [0-9]+\.[0-9]+\.[0-9]+ ]]; then
     exit 1
 fi
 
+# MKSPEC_ABI_REPLACE defines a sed substitution pattern which is applied in setup_cross_compiler
```

Comment:
Looks like `ABI_NAME` is also arm-specific - so `MKSPEC_ABI_REPLACE` should follow the same pattern - being set to `""` inside the case.